intake/views.py

```diff
@@ -2,118 +2,213 @@
 from django.contrib.auth.decorators import login_required
 from IPython import embed
 
+from intake.services import *
+
+import datetime
+
 def index(request):
-    context = {'message': 'randy..............thank you'}
+    context = {}
     return render(request, 'intake/index.html', context)
 
 @login_required
-def mp_threshold_question(request):
+def new_request(request):
+    return render(request, 'intake/new_request.html', {})
+
+@login_required
+def create_request(request):
+    if request.method == 'POST':
+        request = CreateRequest().perform()
+        return redirect('/requests/' + str(request.pk) + '/start')
+
+@login_required
+def mp_threshold_question(request, request_id):
```

Comment:
If somebody goes to a URL with a request id that doesn't exist, should they get redirected back to the start?